### ISSUE TITLE
Conflicting peer dependency: @vue/composition-api@1.0.0-rc.6

### ISSUE THREAD

**Issue by eorenstein1** [protonemedia/inertiajs-tables-laravel-query-builder#6]
Trying to install `@protonemedia/inertiajs-tables-laravel-query-builder` in a Vue3 project results in peer dependency conflict. I was able to install by using the --force option. Below is the full NPM log.

`````
# npm resolution error report

2021-04-12T00:26:11.826Z


Could not resolve dependency:
@protonemedia/inertiajs-tables-laravel-query-builder@"*" from the root project

Conflicting peer dependency: @vue/composition-api@1.0.0-rc.6
node_modules/@vue/composition-api
  peerOptional @vue/composition-api@"^1.0.0-rc.6" from @protonemedia/inertiajs-tables-laravel-query-builder@1.1.0
  node_modules/@protonemedia/inertiajs-tables-laravel-query-builder
    @protonemedia/inertiajs-tables-laravel-query-builder@"*" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

Raw JSON explanation object:

{
  "code": "ERESOLVE",
  "current": null,
  "currentEdge": null,
  "edge": {
    "type": "prod",
    "name": "@protonemedia/inertiajs-tables-laravel-query-builder",
    "spec": "*",
    "error": "MISSING",
    "from": {
      "location": ""
    }
  },
  "peerConflict": {
    "name": "@vue/composition-api",
    "version": "1.0.0-rc.6",
    "whileInstalling": {
      "path": ""
    },
    "location": "node_modules/@vue/composition-api",
    "dependents": [
      {
        "type": "peerOptional",
        "name": "@vue/composition-api",
        "spec": "^1.0.0-rc.6",
        "from": {
          "name": "@protonemedia/inertiajs-tables-laravel-query-builder",
          "version": "1.1.0",
          "whileInstalling": {
            "path": ""
          },
          "location": "node_modules/@protonemedia/inertiajs-tables-laravel-query-builder",
          "dependents": [
            {
              "type": "prod",
              "name": "@protonemedia/inertiajs-tables-laravel-query-builder",
              "spec": "*",
              "from": {
                "location": ""
              }
            }
          ]
        }
      }
    ]
  },
  "strictPeerDeps": false,
  "force": false
}
`````

**Comment by pascalbaljet** [comment]
Thanks for sharing! Do you use Laravel Jetstream + Vue 3 by any chance?

**Comment by eorenstein1** [comment]
Yes, I'm using Larvel Jetstream + Vue 3.

**Comment by pascalbaljet** [comment]
Fixed in v1.1.2
